packaging/centos7/cloud-management.service

```diff
@@ -24,6 +24,7 @@ Description=CloudStack Management Server
 After=syslog.target network.target
 
 [Service]
+UMask=0022
```

Comment:
@rafaelweingartner I'm sorry, I don't intend to be rude to you. Bear in mind, I'm doing this after hours in my free time as you are reviewing in your free time; I'm tired and exhausted and I just want to see things move. I'm embarrassed hence the facepalm.

I'm trying to help you here, I've shared a list of things (in a comment below) that you can find useful and improve on -- kind of reviewing the reviewer, and it's fine with me if you disagree on things I suggested.

This is my experience and view (and the view in not specific to you but reviewers in general) -- In an opensource community, we need to know the kind of expectation we can have from PR authors. For this specific case, it's alright that someone did not understand something, but PR author is not responsible or required to explain how xyz works, in this case how systemd scripts work especially when you can read the documentation. In ACS specific community we take reviews seriously and PR generally get blocked unless outstanding issues get resolved, and in a way that holds the PR author like a hostage where they either fix it the reviewer's way, or reach a compromise or have his contribution rot. This general ends up wasting three people's time -- the author, the release manager (wherever applicable) and the reviewer; and time wasted when something (like this) may become a bigger issue than what was intended in the first place. Lastly, when we're working in globally distributed timezone it adds a drag, momentum is lost and we lose contribution pace or something interest.

To give you a background -- last month, we saw sort of a CloudStack "ice-age" where not a single commit was committed/merged for more than 30 days. I received emails from users asking if the project is dying. I figured what I can do to fix this, I fixed Travis so we've a faster way to check PRs. We also want to be pragmatic to PRs and avoid wasting time, or dragging over issues.